app/controllers/v0/burial_claims_controller.rb

```diff
@@ -2,6 +2,32 @@
 
 module V0
   class BurialClaimsController < ClaimsBaseController
+    def create
+      PensionBurial::TagSentry.tag_sentry
+      claim = claim_class.new(form: filtered_params[:form])
+      unless claim.save
+        StatsD.increment("#{stats_key}.failure")
+        raise Common::Exceptions::ValidationErrors, claim
+      end
+
+      veteran_exists_in_mvi = BipClaims::Service.lookup_veteran_from_claim(claim)
+      claimant_exists_in_mvi = false # TODO: add lookup for claimant based on form data
+
+      if veteran_exists_in_mvi && claimant_exists_in_mvi
+
+      else
+        # TODO: Move call to CentralMail here
+      end
+
+      claim.process_attachments! # upload claim and attachments to Central Mail
+
+      StatsD.increment("#{stats_key}.success") # TODO: potentially log creation of claims with Claims API; veteran/claimant lookup failure
+      Rails.logger.info "ClaimID=#{claim.confirmation_number} Form=#{claim.class::FORM}"
+      validate_session
```

Comment:
Just curious: What leads you to verify that a user is logged in towards the end of a controller action?